js/calendar.js

```diff
@@ -439,16 +441,7 @@ if(!String.prototype.formatNum) {
 
 		data.events = this.getEventsBetween(start, end);
 
-		switch(this.options.view) {
```

Comment:
Can ignore this change. I made it because I wanted to display start and end hours of events on the event-list